src/public_data/index.md

```diff
@@ -0,0 +1,91 @@
+# Public Data
+
+A public dataset is a dataset in [BigQuery][bigquery] which is made available to the general public
+in BigQuery or through our [public HTTP endpoint][public_data_endpoint].
+
+## Accessing Public Data in BigQuery
+
+To access public datasets in BigQuery, a [Google Cloud Platform][gcp] (GCP) account is required. 
+GCP also offers [a free tier][gcp_free] which offers free credits to use and run queries in BigQuery.
```

Comment:
I think it's important to mention that no payment information is needed in order to use BigQuery Sandbox as discussed in https://cloud.google.com/blog/products/data-analytics/query-without-a-credit-card-introducing-bigquery-sandbox

Folks are often used to free tiers coming along with the possibility of being automatically billed for overages.